motor_spec:
    gear_ratio : 213
    wheel_base : 0.150
    wheel_radius : 0.034 
    enc_pulse : 44

use_imu_during_odom_calc:
    use_imu : False
    complementary_filter_coef : 0.5